docs/dev-guides/8-xrpackage-engine-api.md

```diff
@@ -122,7 +169,22 @@ _Removes an XRPackage from the Engine._
 
 **Throws** an `Error` if the given `p` is not a child of this `XRPackageEngine` instance.
 
-## `render`
+## `render(pak, width, height, viewMatrix, projectionMatrix, framebuffer)`
+
+_Performs a render of the engine._
```

Comment:
We could note here that it's useful for rendering things like mirrors.

Reply:
Thanks, updated!